src/dsl.jl

```diff
@@ -682,7 +703,7 @@ end
 # `vars_extracted`: A vector with extracted variables (lhs in pure differential equations only).
 # `dtexpr`: If a differential equation is defined, the default derivative (D ~ Differential(t)) must be defined.
 # `equations`: a vector with the equations provided.
-function read_equations_options(options, variables_declared)
+function read_equations_options(options, variables_declared; requiredec = false)
```

Comment:
Making `require_declaration` is probably not necessary, since these are internal functions (almost) always called using `requiredec` as input?

Reply:
Sorry not understanding, what do you mean by making `require_declaration`?